src/core.cpp

```diff
@@ -72,6 +81,10 @@ namespace Core
 	
 	/** The Address for the Last Round. **/
 	std::string LAST_ROUND_BLOCKFINDER = "2S4PPSznWfPVLtPJNpi8Ly46Wft3wGbayGkhaGzKLVcepmrhKTP";
+
+	unsigned int nConnections = 0;
+	unsigned int nLastBlockFound = 0;
```

Comment:
It's stored in core.cpp (as it is global), but is actually assigned to by the daemon thread (/LLP/daemon.cpp) when the client submits a block.  It is then used by statscollector.cpp